docs/src/network-data.md

```diff
@@ -11,10 +11,14 @@ The network data dictionary structure is roughly as follows:
 ```json
 {
 "name":<string>,
-"version":"2",
 "baseMVA":<float>,
 "source_type":<string>,
-"source_version":<string>,
+"source_version":{
+    "major":<int>,
+    "minor":<int>,
+    "patch":<int>,
+    ...
+},
```

Comment:
This doesn't seem necessary, since we can just use a single string to represent a version. Is this to support some change I'm not aware of having been discussed?

Reply:
Ah, I wasn't aware that it did this. I will make a new Issue to fix it, and will revert the documentation when it gets fixed.